proto/osmosis/gamm/pool-models/balancer/balancerPool.proto

```diff
@@ -102,6 +102,13 @@ message PoolAsset {
     (gogoproto.moretags) = "yaml:\"weight\"",
     (gogoproto.nullable) = false
   ];
+
+  // Weight that is normalized s.t. the sum of all pool assets' weights equals 1. 
+  string normalizedWeight = 3 [
```

Comment:
nit: field names are supposed to be snake_cased. Ref: https://developers.google.com/protocol-buffers/docs/style

```suggestion
  string normalized_weight = 3 [
```